### PR TITLE
Opera

### PR DESCRIPTION
**Pull request by jsa34**
- [X] By placing an `X` in the preceding checkbox, I verify that I have signed the [Contributor License Agreement](https://github.com/SeleniumHQ/docker-selenium/blob/master/CONTRIBUTING.md#contributing-code-to-selenium)

### CODE REVIEW

File: StandaloneOpera/README-short.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Selenium Standalone configured to run Firefox
```
Comment (cgoldberg) [inline StandaloneOpera/README-short.txt]: s/Firefox/Opera/

**Comment by diemol** [comment]
Hi @jsa34, thanks for the PR.

Currently the build is failing:

```
cd ./NodeOpera && ./generate.sh master selenium SeleniumHQ
/bin/sh: 1: ./generate.sh: Permission denied
make: *** [generate_opera] Error 126
```

But more than that, I think we will not merge this PR since Opera has a [low usage](http://gs.statcounter.com/browser-market-share/desktop/worldwide) and it will increase maintenance of this project considerably. 

It occurs to me that you could maintain the Opera images in your fork and we could add in the main README a note pointing users that, so if someone needs Opera, they could refer to your fork.

What do you think @ddavison?

**Comment by ddavison** [comment]
i use Opera as my day to day browser.  i believe i started working on a PR to add it, but stopped for some reason.  IIRC, it was because it's a waste of space to have Opera as it's still Chromium.. I believe there was no added benefit to have Opera when a person could just use Chrome.

**Comment by jsa34** [comment]
Let me know and if you want to include, I can look into the permissions issue on ./generate.sh - not sure why it's behaving like this on Travis, as my local seemed fine...

**Comment by jsa34** [comment]
And thanks all for the feedback 😀

**Comment by diemol** [comment]
Well, I also left this one open for a while to see if people popped in and supported this addition. But no one has expressed any interest or a 👍to support it. 

I really appreciate the effort put into this @jsa34, but I don't see much benefit from it, and it would increase the project maintenance. Unless there are strong arguments in favor of adding it, I will close this PR in the next days.

**Comment by ddavison** [comment]
@diemol perhaps we can add a wiki page regarding Opera.  advising to use Chrome

**Pull request closed without merging**